src/inverseRobot.js

```diff
@@ -23,8 +23,24 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
   // write code here
+  const inversedRobot = {};
+  const robotKey = Object.keys(robot);
```

Comment:
one key or array of keys?
```suggestion
  const robotKeys = Object.keys(robot);
```